src/main/scala/nl.knaw.dans.easy.authinfo/components/FileItem.scala

```diff
@@ -28,7 +28,7 @@ import org.json4s.JsonDSL._
 
 import scala.collection.JavaConverters._
 
-case class FileItem(id: UUID, path: Path, owner: String, rights: FileRights, dateAvailable: String) extends DebugEnhancedLogging {
+case class FileItem(id: UUID, path: String, owner: String, rights: FileRights, dateAvailable: String) extends DebugEnhancedLogging {
 
   val solrLiterals: CacheLiterals = Seq(
     ("id", s"$id/$path"),
```

Comment:
So... something like
```scala
("id", s"$id/${path.escapePath}"),
```

Reply:
shouldn't the `/` between id and path be escaped too?

Reply:
@jo-pol : the `/` should **definetely not** be escaped. The idea of escapePath is to escape the components op the path.

Reply:
@rvanheest : no, the `path` variable is already escaped.